src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/RetryLogic/RetryLogicTestHelper.cs

```diff
@@ -0,0 +1,227 @@
+                    40143,  // The service has encountered an error processing your request. Please try again.
+                    40613,  // Database '%.*ls' on server '%.*ls' is not currently available. Please retry the connection later. If the problem persists, contact customer support, and provide them the session tracing ID of '%.*ls'.
+                    40501,  // The service is currently busy. Retry the request after 10 seconds. Incident ID: %ls. Code: %d.
+                    40540,  // The service has encountered an error processing your request. Please try again.
+                    40197,  // The service has encountered an error processing your request. Please try again. Error code %d.
+                    10929,  // Resource ID: %d. The %s minimum guarantee is %d, maximum limit is %d and the current usage for the database is %d. However, the server is currently too busy to support requests greater than %d for this database. For more information, see http://go.microsoft.com/fwlink/?LinkId=267637. Otherwise, please try again later.
+                    10928,  // Resource ID: %d. The %s limit for the database is %d and has been reached. For more information, see http://go.microsoft.com/fwlink/?LinkId=267637.
+                    10060,  // An error has occurred while establishing a connection to the server. When connecting to SQL Server, this failure may be caused by the fact that under the default settings SQL Server does not allow remote connections. (provider: TCP Provider, error: 0 - A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.) (Microsoft SQL Server, Error: 10060)
+                    10054,  // The data value for one or more columns overflowed the type used by the provider.
+                    10053,  // Could not convert the data value due to reasons other than sign mismatch or overflow.
+                    997,    // A connection was successfully established with the server, but then an error occurred during the login process. (provider: Named Pipes Provider, error: 0 - Overlapped I/O operation is in progress)
+                    233,    // A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.) (Microsoft SQL Server, Error: 233)
+                    64,
+                    20,
+                    0,
+                    -2,     // Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.
+                    207    // invalid column name
```

Comment:
Is "207" retriable error?

Reply:
This error number and the other error numbers like 102, 2812 have been added just for the test purpose to assess the Configurable Retry Logic program flow.

Reply:
Ok, are we testing on these too?

Reply:
Yes, you can find them [here](https://github.com/dotnet/SqlClient/pull/693/files#diff-183589ef7293b57f73bbc3a2a517befb18051e8bf89f16d17b6d5f8609135051R136) that is used in the SqlCommand test [RetryExecuteUnauthorizedSqlStatementDML](https://github.com/dotnet/SqlClient/blob/9cd227b0e2d5517f59e21a66059409899dd0a223/src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/RetryLogic/SqlCommandCRLTest.cs#L176).